test/function/samples/empty-string-as-module-name/main.js

```diff
@@ -0,0 +1,2 @@
+import { importee } from '';
+assert.strictEqual(importee, 'xxx');
```

Comment:
When the test is run, Node's `assert` library is always available as a global variable. The best practice is to add assertions inline into the code. If you change for instance the `xxx`, the test will fail.

Reply:
In our situation, having the `xxx` as importee is definitely a clear indication that everything worked as expected.